app/src/main/java/com/pajato/android/gamechat/database/DBUtils.java

```diff
@@ -123,7 +123,7 @@ public String getHandlerName(@NonNull final String base, @NonNull final String m
         return String.format(Locale.US, "%s{%s}", base, modelName);
     }
 
-    /** Return a, poissibly null, system resource. */
+    /** Return a, possibly null, system resource. */
```

Comment:
It occurs to me that I should start going through the GitHub diffs (in addition to the Emacs diffs) to catch these mistakes.